bindata/network/ovn-kubernetes/managed/common/004-config.yaml

```diff
@@ -127,6 +127,9 @@ data:
 {{- if (index . "V6JoinSubnet") }}
     v6-join-subnet="{{.V6JoinSubnet}}"
 {{- end }}
+
+    [logging]
+    libovsdblogfile=/var/log/ovnkube/libovsdb.log
```

Comment:
@kyrtapz do you have an opinion on how to mimic the size and rotation of logs that kubectl uses? We can specify them here using the following flags (https://github.com/openshift/ovn-kubernetes/blob/f05d79a6080efe659ad6445ccd2a5636cfabf172/go-controller/pkg/config/config.go#L78-L80):
- logfile-maxsize
- logfile-maxbackups
- logfile-maxage

Reply:
As we discussed in meeting, let's disable max-age and use the other default values.

https://github.com/natefinch/lumberjack/blob/4cb27fcfbb0f35cb48c542c5ea80b7c1d18933d0/lumberjack.go#L89-L94
```
	// MaxAge is the maximum number of days to retain old log files based on the
	// timestamp encoded in their filename.  Note that a day is defined as 24
	// hours and may not exactly correspond to calendar days due to daylight
	// savings, leap seconds, etc. The default is not to remove old log files
	// based on age.
	MaxAge int `json:"maxage" yaml:"maxage"`
```